### DISCUSSION TITLE
MongoDB Support

### DISCUSSION THREAD

**Discussion by kramer1007** [microsoft/azuredatastudio#21005] [Ideas]
Add MongoDB support to Azure Data Studio

**Comment by Charles-Gagnon** [comment]
From https://github.com/microsoft/azuredatastudio/issues/5558#issuecomment-1279795297 : 

> There is an extension for COSMOS DB in Azure Data Studio: https://github.com/microsoft/azuredatastudio/issues/4916#issuecomment-1267512387.
> 
> Please open issue in the extension team's own repository for feature requests/bugs.

https://github.com/Azure/azure-cosmosdb-ads-extension

**Comment by erinstellato-ms** [comment]
There is also an extension for MongoDB Atlas, introduced in 1.42 (January 2023 release).

**Reply by kramer1007** [reply]
Nice, I'll check it out.
